concordium-std/src/prims.rs

```diff
@@ -223,6 +223,8 @@ extern "C" {
 
     // Getters for the chain meta data
     /// Slot time (in milliseconds) from chain meta data
+    /// Note that this is a Timestamp (unix timestamp with millisecond
+    /// precision).
```

Comment:
The comment just above already states it is milliseconds.

Reply:
Yea.. I removed one of the 'milliseconds' :)